internal_metrics/dimensions.yaml

```diff
@@ -0,0 +1,171 @@
+  example: ip-192-168-110-244.ec2.internal
+
+c_type:
+  brief: Client type
+  description: Client types are numbers desginating different client types. Different types are kernel(1), cloud(2), k8s(3), ingest(4), matching(5), aggregation(6), liveness_probe (7), readiness_probe(8).
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.clock_offset_ns, ebpf_net.collector_health
+  example: 1
+
+cloud:
+  brief: Cloud type
+  description: Cloud provider type where network explorer is installed.
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.collector_health
+  example: 1
+
+detail:
+  brief: Detail
+  description: Detail status of a metric.
+  associated_metrics: ebpf_net.collector_health
+  example: 0
+
+env:
+  brief: environment
+  description: environment where network explorer was installed.
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.bpf_log, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.collector_health
+  example: network-explorer-staging.
+
+error:
+  brief: error type
+  description: error type
+  associated_metrics: ebpf_net.pipeline_message_error, ebpf_net.entrypoint_info
+  example: fetched/unknown
+
+field:
+  brief: Field name
+  description: Field name.
+  associated_metrics: ebpf_net.bpf_log
+  example: pod_name, id, ip, az etc
+
+filename:
+  brief: Filename of the source code file.
+  description: File name of the source code file.
+  associated_metrics: ebpf_net.codetiming_min_ns, ebpf_net.codetiming_max_ns, ebpf_net.codetiming_sum_ns, ebpf_net.codetiming_avg_ns, ebpf_net.codetiming_count
+  example: agg_core.cc
+
+index:
+  brief: Index count
+  description: Number of Index objects. Index objects contain span pools for all span types.
+  associated_metrics: ebpf_net.codetiming_min_ns, ebpf_net.codetiming_max_ns, ebpf_net.codetiming_sum_ns, ebpf_net.codetiming_avg_ns, ebpf_net.codetiming_count
+  example: 5
+
+id:
+  brief: Id
+  description: Id is the node identifier.
+  associated_metrics: ebpf_net.time_since_last_message_ns, ebpf_net.pipeline_message_error, ebpf_net.message, ebpf_net.entrypoint_info, ebpf_net.collector_log_count, ebpf_net.client_handle_pool, ebpf_net.client_handle_pool_fraction, ebpf_net.clock_offset_ns, ebpf_net.collector_health
+  example: us-east-1b
```

Comment:
This example looks like an az, not a node identifier

Reply:
Good catch!